### PR TITLE
Test order of convergence

### PR DESCRIPTION
**Pull request by dominikbell**
- Add a test for the order of convergence for the Arakawa scheme with different orders and boundary conditions for the two problems of constant advection and vortex advection
- Rewrite visual tests in better coding style and without splines

### CODE REVIEW

File: pygyro/advection/visual_test_advection.py [hunk 1]
```diff
@@ -337,50 +337,46 @@ def test_poloidalAdvectionArakawa_vortex():
     """
     TODO
     """
-    npts = [30, 20]
-    eta_vals = [np.linspace(0, 20, npts[1], endpoint=False), np.linspace(0, 2*pi, npts[0], endpoint=False),
-                np.linspace(0, 1, 4), np.linspace(0, 1, 4)]
+    N_theta = 80
```
Comment (FrederikSchnack) [inline pygyro/advection/visual_test_advection.py:340]: I would suggest to keep the same `npts`

Comment (dominikbell) [reply pygyro/advection/visual_test_advection.py]: done

**Review by FrederikSchnack — APPROVED** [review]
:+1:
